app/controllers/comments_controller.rb

```diff
@@ -657,7 +657,8 @@ def redirect_to_all_comments(commentable, options = {})
                   delete_comment_id: options[:delete_comment_id],
                   view_full_work: options[:view_full_work],
                   anchor: options[:anchor],
-                  page: options[:page]
+                  page: options[:page],
+                  only_path: true
```

Comment:
Conflicted about this.

On one hand, it does fix every issue I had in tests about being redirect to the wrong domain (example.com).

On the other, changing such a critical piece of code just so tests pass trigger every warning in my brain.

Reply:
Yeah, I'm normally not a fan of that either, and the alarm bells are screaming. But ignoring my knee-jerk "let's not do this," I can't think of a reason it _shouldn't_ be `only_path: true`. We definitely prefer paths to URLs. I think the only reason we hadn't already changed it is [we do something in nginx that means this wasn't actively causing issues](https://otwarchive.atlassian.net/browse/AO3-4630) when we were supporting both HTTP and HTTPS. So this is probably fine.